app/src/main/java/org/oppia/app/player/exploration/ExplorationActivityPresenter.kt

```diff
@@ -228,7 +235,7 @@ class ExplorationActivityPresenter @Inject constructor(
     explorationLiveData.observe(
       activity,
       Observer<Exploration> {
-        explorationToolbar.title = it.title
+        explorationToolbarTitle.text = it.title
```

Comment:
You can hard-code string here something like:
`Long toolbar title for testing purposes`